web/src/ShipClientGQL.js

```diff
@@ -17,9 +21,16 @@ export function ShipClientGQL(graphqlEndpoint, restEndpoint, tokenFunction, fetc
   const stateLink = withClientState({
     cache
   });
+  
```

Comment:
There was an error from Apollo: 

> [Network error]: Error: only absolute urls are supported

So I have to check first is url apsolute or relative.